physics/test/CMakeLists.txt

```diff
@@ -175,3 +175,24 @@ target_include_directories(testConstantOcn PRIVATE
     "${ModulesDir}"
     )
 target_link_libraries(testConstantOcn PRIVATE Boost::program_options Boost::log doctest::doctest Eigen3::Eigen)
+
+add_executable(testSlabOcn
```

Comment:
Well 'ocn' and 'atm' are fairly common abbreviations for 'ocean' and 'atmosphere' in geophysics (or for 'oceanic' and 'atmospheric'). Here it would be consistent with `ConfiguredAtm` and `ConstantOcn` in the same CMake file.